apps/cnspec/cmd/run.go

```diff
@@ -17,8 +18,147 @@ func init() {
 
 var execCmd = builder.NewProviderCommand(builder.CommandOpts{
 	Use:   "run",
-	Short: "Run an MQL query",
+	Short: "Run an MQL query.",
 	Long:  `Run an MQL query on the CLI and displays its results.`,
+	Docs: common.CommandsDocs{
+		Entries: map[string]common.CommandDocsEntry{
+			"local": {
+				Short: "Run an MQL query against your local system.",
+			},
+			"mock": {
+				Short: "Run an MQL query against a mock target (a simulated asset).",
+			},
+			"vagrant": {
+				Short: "Run an MQL query against a Vagrant host.",
+			},
+			"terraform": {
+				Short: "Run an MQL query against Terraform HCL (files.tf and directories), plan files (json), and state files (json).",
+			},
+			"ssh": {
+				Short: "Run an MQL query against an SSH target.",
+			},
+			"winrm": {
+				Short: "Run an MQL query against a WinRM target.",
+			},
+			"container": {
+				Short: "Run an MQL query against a container, image, or registry.",
+			},
+			"container-image": {
+				Short: "Run an MQL query against a container image.",
+			},
+			"container-tar": {
+				Short: "Run an MQL query against an OCI container image from a tar file.",
+			},
+			"container-registry": {
+				Short: "Run an MQL query against a container registry.",
+			},
+			"docker": {
+				Short: "Run an MQL query against a Docker container or image.",
+			},
+			"docker-container": {
+				Short: "Run an MQL query against a Docker container.",
+			},
+			"docker-image": {
+				Short: "Run an MQL query against a Docker image.",
+			},
+			"kubernetes": {
+				Short: "Run an MQL query against a Kubernetes cluster or local manifest file(s).",
+			},
+			"aws": {
+				Short: "Run an MQL query against an AWS account or instance.",
+			},
+			"aws-ec2": {
+				Short: "Run an MQL query against an AWS instance using one of the available connectors.",
+			},
+			"aws-ec2-connect": {
+				Short: "Run an MQL query against an AWS instance using EC2 Instance Connect.",
+			},
+			"aws-ec2-ebs-instance": {
+				Short: "Run an MQL query against an AWS instance using an EBS volume scan. This requires an AWS host.",
+			},
+			"aws-ec2-ebs-volume": {
+				Short: "Run an MQL query against a specific AWS volume using an EBS volume scan. This requires an AWS host.",
+			},
+			"aws-ec2-ebs-snapshot": {
+				Short: "Run an MQL query against a specific AWS snapshot using an EBS volume scan. This requires an AWS host.",
+			},
+			"aws-ec2-ssm": {
+				Short: "Run an MQL query against an AWS instance using the AWS Systems Manager to connect.",
+			},
+			"azure": {
+				Short: "Run an MQL query against a Microsoft Azure subscription or virtual machine.",
+			},
+			"gcp": {
+				Short: "Run an MQL query against a Google Cloud Platform (GCP) organization, project or folder.",
```

Comment:
I think, this is missing:
```suggestion
				Short: "Run an MQL query against a Google Cloud Platform (GCP) organization, project, or folder.",
```